code/integration/Main/SoftwareSerial_mod.h

```diff
@@ -0,0 +1,123 @@
+/*
+SoftwareSerial.h (formerly NewSoftSerial.h) - 
+Multi-instance software serial library for Arduino/Wiring
+-- Interrupt-driven receive and other improvements by ladyada
+   (http://ladyada.net)
+-- Tuning, circular buffer, derivation from class Print/Stream,
+   multi-instance support, porting to 8MHz processors,
+   various optimizations, PROGMEM delay tables, inverse logic and 
+   direct port writing by Mikal Hart (http://www.arduiniana.org)
+-- Pin change interrupt macros by Paul Stoffregen (http://www.pjrc.com)
+-- 20MHz processor support by Garrett Mace (http://www.macetech.com)
+-- ATmega1280/2560 support by Brett Hagman (http://www.roguerobotics.com/)
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 2.1 of the License, or (at your option) any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+The latest version of this library can always be found at
+http://arduiniana.org.
+*/
+
+#ifndef SoftwareSerial_h
+#define SoftwareSerial_h
+
+#include <inttypes.h>
+#include <Stream.h>
+
+/******************************************************************************
+* Definitions
+******************************************************************************/
+
+#ifndef _SS_MAX_RX_BUFF
+#define _SS_MAX_RX_BUFF 150 // RX buffer size
```

Comment:
Ich weiß nicht ob uns das wirklich hilft (oder nicht sogar Speicher weg nimmt, den wir eigtl. bräuchten). Bei ankommenden Nachrichten hatten wir bisher noch nie Probleme mit dem 64 Byte Buffer. Ich kenne nur von zwei Fälle von SW Serial Überläufen, einmal wenn der Delay noch aktiv ist und man in dieser Zeit sendet und Florian hat mal einen Überlauf hinbekommen, indem er sich schnell hintereinander an und abgemeldet hat (wobei ich nicht verstehe warum des dabei zum Überlauf kommt). 
Was die ausgehenden Nachrichten angehen meine ich braucht es auch keinen (zusätzlichen) Buffer.

Aber da wir jetzt eh Schluss machen mit Speicheroptimierung, können wir es gerne so lassen.